web/index.html

```diff
@@ -71,7 +71,6 @@
     <!-- link.prod --> <!-- <link type="text/css" rel="Stylesheet" href="ext/image/jcrop-0.9.12/jquery.Jcrop.min.css"> -->
 
                             <link type="text/css" rel="Stylesheet" href="ext/ui/noUiSlider-6.1.10/jquery.nouislider.css">
-                            <link type="text/css" rel="Stylesheet" href="ext/map/openlayers-3.17.1.wv1/ol.css">
```

Comment:
@Benjaki2 It looks like we *were* actually using ol.css before. If for nothing else, it adds the blue outline to a shift+click selection that @minniewong found was missing. But I suspect that if we're missing that, there are likely other pieces we're missing too. Let's add this back in.

Reply:
@localjo interesting. Wonder when/why we pulled it out. In that case we should throw the `ol.debug.css` in for development.

Reply:
I pulled it out in my Event Markers PR because you said we weren't using it before; https://github.com/nasa-gibs/worldview/pull/463#discussion_r139461354

Reply:
Ahh It looked like it was an addition in the PR.

Reply:
Nope, we had just been using the wrong version before, so I replaced it with the correct one;

https://github.com/nasa-gibs/worldview/pull/463/commits/f88dcb6e0e9391ea6fb3b5b58afba447ed340bf8#diff-d23fe9001e474c96419c8f2519cbef67L74

But then I forgot that I moved it and thought maybe I added it by mistake. Anyway, it's all fixed now.